src/root/views/FieldReportForm/ContextFields/index.tsx

```diff
@@ -223,67 +242,42 @@ function ContextFields(props: Props) {
       >
         <table cellSpacing={0} cellPadding={0}>
           <tbody>
+           
            <tr>
-             <td colSpan={5}>
-                <SearchSelectInput
-                  label={strings.fieldReportFormTitleSelectLabel}
-                  placeholder={strings.fieldReportFormTitleSelectPlaceholder}
-                  name={"event" as const}
-                  value={value.event}
-                  onChange={onValueChange}
-                  loadOptions={fetchEventsFromApi}
-                  initialOptions={initialEventOptions as SearchSelectOption[]}
+           {
+              reportId === undefined ? (
+                <>
+              <td style={{width: '35%'}} >
+                <TextInput
+                  style={{ backgroundColor: '#E0DDDD', borderRadius: 0, padding:'offset' }}
+                  label='prefix '//{strings.fieldReportFormCountryLabel}
+                  placeholder=""
+                  name="pref2"
+                  //value={(countryIsoOptions.find(x=> x.value === value.country)?.label + ': ' +  disasterTypeOptions.find(x=>x.value === value.dtype)?.label + ' -' + value.start_date?.substring(0,7)).replaceAll('undefined',' ' )}
+                  value={value.country !== undefined && value.dtype !== undefined && value.start_date !== undefined ? countryIsoOptions.find(x=> x.value === value.country)?.label + ': ' +  disasterTypeOptions.find(x=>x.value === value.dtype)?.label + ' -' + value.start_date?.substring(0,7) : ' '}
```

Comment:
@vdeak this would be really nice to move to a function or so and / or use string formatting instead of `+` for concatenation - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals . We can do this post-release, but please make sure this is done? This code is quite hard to read and splitting it into a function with some comments and also using template literals should help.